Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -238,6 +238,9 @@
         return API::Data::create(std::span<const uint8_t> { });
     }
 
+    if ([cocoaPath hasPrefix:@"symbol:"])
+        return makeUnexpected(nullptr);
```

Comment:
Can `constexpr auto symbolURLScheme = "symbol:"_s;` be put somewhere that it can be used here too?

Reply:
Yeah :(